Automattic/jetpack
# E2E Tests: better slack logging

#### Changes proposed in this Pull Request:

* Introduce new `winston` logger
* Replace `console.log` calls with new logger
* Update slack reporting to post all the build logs into a single thread

#### Testing instructions:






* Check CI E2E build. it should not debug logs anymore
* Check Jetpack e2e notif channel, and find the build output

#### Proposed changelog entry for your changes:

* n/a
